src/RestClient/endpoints/V3/Themes/ThemeActions.ts

```diff
@@ -0,0 +1,34 @@
+import { AxiosInstance } from 'axios';
+
+import { themesPath } from './Themes';
+import type { ThemeAction } from './types';
+
+class ThemeActions {
+  private client: AxiosInstance;
+
+  constructor(client: AxiosInstance) {
+    this.client = client;
+  }
+
+  /**
+   * Downloads a stores Theme.
+   *
+   * @param uuid A valid store Theme UUID
+   * @returns Promise resolving to a single theme
+   */
+  get(uuid: string): ThemeAction['GetResponse'] {
```

Comment:
This method does take a UUID as an identifier, but it also looks like it takes an optional JSON body (that's why this method is a POST method) with a property, "which": https://developer.bigcommerce.com/api-reference/24d4f462e909e-download-a-theme
Please make sure to add another parameter expecting that schema